src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+
+Button.propTypes = {
+  children: PropTypes.node,
+  isActive: PropTypes.bool,
+  onClick: PropTypes.func,
+};
+
+export default function Button(props) {
+  return <ButtonStyled {...props} />;
+}
+
+const ButtonStyled = styled.button`
+  border-style: none;
+  font-family: Helvetica Neue;
```

Comment:
Either use your variable or there is no need to use it at all because you set the font-family in the `body`. If it's needed, then use `font-family: inherit;`